src/main/java/com/wix/mysql/config/extract/NopNaming.java

```diff
@@ -4,6 +4,18 @@
 
 public class NopNaming implements ITempNaming {
 
+    private final String basePath;
+
+    public NopNaming() {
```

Comment:
@viliusl you basically killed NopNaming, create another implementation and call it BasaPathAppendingNaming or something like that, nop means no operation....